### PR TITLE
Make a complete node table

### PR DESCRIPTION
**Pull request by alec-deason**
This fills the node table with the complete gbd hierarchy for the round the model is associated with and deals with converting location_ids into node_ids and back. This should be examined after my two currently outstanding PRs because it assumes their code.

### CODE REVIEW

File: src/cascade/dismod/db/wrapper.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 """
 import logging
 
+from textwrap import dedent
```
Comment (adolgert) [inline src/cascade/dismod/db/wrapper.py]: I like this module. Nice.

File: src/cascade/dismod/serialize.py [hunk 2]
```diff
@@ -147,25 +137,35 @@ def make_log_table():
...
+
+
+def make_data_table(context, node_table):
     total_data = []
     if context.input_data.observations is not None:
-        total_data.append(observations_to_data(context.input_data.observations))
+        total_data.append(observations_to_data(context.input_data.observations, node_table))
     if context.input_data.constraints is not None:
-        total_data.append(observations_to_data(context.input_data.constraints, hold_out=1))
+        total_data.append(observations_to_data(context.input_data.constraints, node_table, hold_out=1))
```
Comment (adolgert) [inline src/cascade/dismod/serialize.py]: Constraints are smoothings, not observations. Will ask on dismod-dev.

**Comment by alec-deason** [comment]
I learned it from Mike

________________________________

**Comment by alec-deason** [comment]
Greg's idea. He knows how the tool works.

________________________________

**Pull request merged**